tests/unit/h/services/group_members_test.py

```diff
@@ -88,7 +85,7 @@ def test_it_does_not_remove_existing_members(
 class TestUpdateMembers:
     def test_it_adds_users_in_userids(self, factories, group_members_service):
         group = factories.OpenGroup()  # no members at outset
-        new_members = [factories.User(), factories.User()]
+        new_members = (factories.User(), factories.User())
```

Comment:
`new_members` will be compared to `group.members` in an `assert` below and `group.members` is now a tuple so `new_members` also needs to be a tuple now or the assert fails